pkg/volume/iscsi/iscsi_util.go

```diff
@@ -299,6 +299,18 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) (string, error) {
 
 	//Make sure we use a valid devicepath to find mpio device.
 	devicePath = devicePaths[0]
+	for _, path := range devicePaths {
```

Comment:
why move this up?

Reply:
Because devicePath will be returned at line 323, devicePath should be updated beforehand.